src/plugins/kibana/public/settings/sections/indices/directives/install_filebeat_step.html

```diff
@@ -0,0 +1,34 @@
+<div class="wizard-step-title">
+  <h3>Follow these instructions to install Filebeat</h3>
+  Now that you've got a fresh pipeline and index pattern, let's throw some data at it!
```

Comment:
should we be more explicit about what we just indexed into Elasticsearch? For example, it would be useful to know what the pipeline was indexed as (pipeline name) and instructions on where to find it